ui/components/ui/icon/icon-eye.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const IconEye = ({
+  size = 24,
+  color = 'currentColor',
+  ariaLabel,
+  className,
+}) => (
+  <svg
+    width={size}
+    height={size}
+    fill={color}
+    className={className}
+    aria-label={ariaLabel}
+    xmlns="http://www.w3.org/2000/svg"
+    viewBox="0 0 512 512"
+  >
+    <path d="m444 247c-36-71-108-119-190-119-82 0-154 48-190 119-1 2-2 6-2 9 0 4 1 7 2 10 36 71 108 118 190 118 82 0 154-47 190-118 1-3 2-6 2-9 0-4-1-8-2-10z m-190 105c-53 0-96-43-96-96 0-53 43-96 96-96 53 0 96 43 96 96l0 1c0 52-43 96-96 96z m0-160c-6 1-12 1-17 3 4 5 6 12 6 19 0 17-14 31-31 31-8 0-14-2-20-6-1 6-2 12-2 17 0 35 29 64 64 64 36 0 64-29 64-64 0-35-28-63-64-63z" />
```

Comment:
Where did this SVG come from? Is this fontawesome?

Reply:
Yes, I just grabbed it from the Figma file but I wasn't sure where it was from. I think ideally we are using svgs over font icons. But do you think this could be breaking a usage agreement?

Reply:
If it's from the "FontAwesome Free" pack, this is allowed under their creative commons license. We're allowed to embed it and change it however we like. But it's dangerous to embed IP we don't own directly in our product without documenting what it is or where it came from. _Minimally_ we should document what it is and where it came from. That way we can be assured that it's allowed.

Reply:
I'd strongly recommend against _ever_ copying an SVG directly from Figma. Our Figma designs have included "FontAwesome Pro" icons in the past (which we cannot use, because our usage is incompatible with their license, they won't even let us pay for it), and they have contained other proprietary icons from other sources.  Big no-no. Please don't ever do this ,and speak up if you ever see somebody else doing it.

Reply:
OK thanks. That's good to know. This should be flagged at the design level or at least before design handover to dev. Is this common knowledge in design? Might be worth reminding them. As for using font icons they can render unexpectedly in different browsers so we should be pushing for svg icons. I've seen a few features using svgs go into the codebase with svgs being pulled from Figma so I think we should definitely flag this in the right channels.

Reply:
We do have all of the SVG icons as a dependency (`@fortawesome/fontawesome-free`, see the `svgs` directory). These are the official SVGs. We could copy them direct from there and leave a comment saying where they came from (including the version). Or we could use the FontAwesome React library, where they've already packaged each SVG as a component. See here for details: https://fontawesome.com/v6/docs/web/use-with/react/

Reply:
I copied over the official SVGs and added a comment explaining where we got them